source/StigData/Processed/Office-365ProPlus-2.3.xml

```diff
@@ -519,9 +520,9 @@ Use the Windows Registry Editor to navigate to the following key:
 
 HKCU\software\policies\microsoft\office\16.0\common\trustcenter
 
-If the value for trustbar is set to REG_DWORD = 0, this is not a finding.</RawString>
-      <ValueData>0</ValueData>
-      <ValueName>trustbar</ValueName>
+If the value automationsecuritypublisher is REG_DWORD = 2, this is not a finding.</RawString>
+      <ValueData>2</ValueData>
+      <ValueName>automationsecuritypublisher</ValueName>
```

Comment:
In the same way as the comment above, this rule should be evaluated as well, since it changed.

Reply:
Rule appears to be correct:

![proplus523](https://user-images.githubusercontent.com/20539446/128450937-4ffd405a-6107-49b6-b9b1-213a3db379b6.JPG)